src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -226,7 +226,9 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 widgetPlaceholder.getChildren().add(countryNoteListPanel.getRoot());
             }
 
-            if (commandResult.isResetWidget()) {
+            if (commandResult.shouldResetWidget()) {
+                widgetPlaceholder.getChildren().clear();
+                widgetPlaceholder.getChildren().add(widgetViewBox.getRoot());
```

Comment:
Bugfix for when `country note view` is called followed by `client list`, which doesn't reset the widget display viewbox back to default, even though it should.

Reply:
👍